content/enterprise_influxdb/v1.9/introduction/install-and-deploy/installation/meta_node_installation.md

```diff
@@ -257,7 +257,5 @@ Note that your cluster must have at least three meta nodes.
 If you do not see your meta nodes in the output, retry adding them to
 the cluster.
 
-Once your meta nodes are part of your cluster move on to [the next steps to
-set up your data nodes](/enterprise_influxdb/v1.9/install-and-deploy/installation/data_node_installation/).
-Please do not continue to the next steps if your meta nodes are not part of the
-cluster.
+Once your meta nodes are part of your cluster, move on to [install data nodes](/enterprise_influxdb/v1.9/install-and-deploy/installation/data_node_installation/).
+Please do not continue to the next steps if your meta nodes are not part of the cluster.
```

Comment:
```suggestion
Do not continue if your meta nodes are not part of the cluster.
```

Reply:
Link to instruction on how to add meta nodes to cluster

Reply:
I think we can remove this line.